packages/transport/README.md

```diff
@@ -21,12 +22,98 @@ npm install @ucanto/transport
 ```
 
 ## Example Usage
-```ts
-import * as CAR from '@ucanto/transport/car';
-import * as CBOR from '@ucanto/transport/cbor';
+```js
+import * as HTTP from '@ucanto/transport/http'
+import { CAR } from '@ucanto/transport'
+import { ed25519 } from '@ucanto/principal'
+import { invoke, Message, Receipt } from '@ucanto/core'
 
-const encoded = CAR.encode({ invocations: [] });
-const decoded = CBOR.decode(encoded);
+const service = ed25519.parse(process.env.SERVICE_ID)
+const issuer = ed25519.parse(process.env.CLIENT_KEYPAIR)
+
+// Mock fetch that simulates a UCAN service
+const mockFetch = async (url, init) => {
```

Comment:
Again, I'd save this for a runnable example in the repo, or better yet a runnable example with an actual server. Mocking the server like this is not great even for tests as it bypasses all delegation chain validation.

Note: in tests, you don't even need a HTTP server running - a Ucanto server _is_ a channel, so you can pass it to the client as the channel to use.